### ISSUE TITLE
cscanpst office 2010 32 with russian interface

### ISSUE THREAD

**Issue by viktor495** [bigjosh/CSCANPST#12]
Thanks for script @bigjosh, i have 1TB of psts which are needed to be scanned rofl ! 
Just sharing how it worked on my side.
Here i have only 2010 office, and it was only russian edition.

main:
I found some lang pack: en_office_language_pack_2010_x86_x64_dvd_516538.iso
Installed it, in Word -> Settings changed interface language to english. Now SCANPST.EXE was on english.
And also in Windows i have set default language to english cause cscanpst uses hotkeys like "Alt+S" which are language sensitive ....

tldr:
I got some "SCANPST.EXE;SCNPST32.DLL;SCNPST64.DLL;OLMAPI32.DLL" files from english version of office (they were of course not exactly the same version as in my russian installed edition, a little bit older ). But for some reason new english SCANPST.EXE still has been opening with russian interface.

Ahh, bat in example is good. Accidentally broke something in it, but now realised that it was my mistake. (what a wild for goto syntax wuh ).

**Comment by bigjosh** [comment]
Thank you for sharing, this will save time for others trying to do this in the future!
